src/mypage/components/MypageMainMenu.tsx

```diff
@@ -1,6 +1,12 @@
 import { useRouter } from "next/router";
 import { useModal } from "@shared/hook";
 
+interface MainMenu {
+  value: string;
+  style: string;
+  onClick?: () => void;
```

Comment:
```suggestion
   onClick?: VoidFunction;
```

[Lv1] () => void; 형식은 VoidFunction으로 표현가능합니다!